cmd/xl-v1-object.go

```diff
@@ -546,7 +546,7 @@ func (xl xlObjects) putObject(ctx context.Context, bucket string, object string,
 	// Delete temporary object in the event of failure.
 	// If PutObject succeeded there would be no temporary
 	// object to delete.
-	defer xl.deleteObject(ctx, minioMetaTmpBucket, tempObj)
+	defer xl.deleteObject(ctx, minioMetaTmpBucket, tempObj, len(xl.getDisks())/2+1, false)
```

Comment:
we should use `writeQuorum` computed few lines above instead of `len(xl.getDisks())/2+1`